modules/packages/RangeChunk.chpl

```diff
@@ -84,6 +84,43 @@ module RangeChunk {
       else start..end;
   }
 
+  /*
+    Iterates through a range ``r``, which is blocked up in repeating ```numInds```
+    blocks of size ```blockSize```. Blocks are indexed from 0..numInds-1. 
+    Emits block at index ```idx``` in a cyclic manner.
+  */
+  iter blockCyclicChunks(r: range(?t, boundedType=BoundedRangeType.bounded, ?strided), blockSize: integral, idx: integral, numInds: integral){
+    if (idx >= numInds) then
+      halt("Parameter idx must be < numInds because blocks are indexed from 0..numInds-1");
+
+    if (blockSize <= 0) then
+      halt("blockSize must a positive number");
+
+    if (numInds <= 0) then
+      halt("numInds must be a positive number");
+
+    var range_stride = r.stride;
```

Comment:
The rest of this file uses camelCase instead of underscores in variable names. We're generally moving toward not using underscores in names throughout the modules.